hub/integrations/pytorch/common.py

```diff
@@ -0,0 +1,21 @@
+from hub.util.iterable_ordered_dict import IterableOrderedDict
+
+
+def _collate_fn(batch):
```

Comment:
docstring and can you add static typing for `batch`?

Reply:
Types for batch and return value depends on whatever pytorch supports. This is just a wrapper on top of pytorch's default collate funtions: https://github.com/pytorch/pytorch/blob/master/torch/utils/data/_utils/collate.py